lib/utility/parent-port.ts

```diff
@@ -3,27 +3,28 @@ import { MessagePortMain } from '@electron/internal/browser/message-port-main';
 const { createParentPort } = process._linkedBinding('electron_utility_parent_port');
 
 export class ParentPort extends EventEmitter {
-  #port: any
+  #port: ParentPort
   constructor () {
     super();
     this.#port = createParentPort();
-    this.#port.emit = (channel: string, event: {ports: any[]}) => {
+    this.#port.emit = (channel: string | symbol, event: {ports: any[]}) => {
       if (channel === 'message') {
         event = { ...event, ports: event.ports.map(p => new MessagePortMain(p)) };
       }
       this.emit(channel, event);
+      return false;
     };
   }
 
-  start () {
-    return this.#port.start();
+  start () : void {
```

Comment:
I'm surprised this doesn't lint to
```suggestion
  start (): void {
```

Reply:
We don't have `@typescript-eslint/type-annotation-spacing` enabled.